bin/strata-client/src/rpc_server.rs

```diff
@@ -105,6 +108,44 @@ fn conv_blk_header_to_rpc(blk_header: &impl L2Header) -> RpcBlockHeader {
 
 #[async_trait]
 impl<D: Database + Send + Sync + 'static> StrataApiServer for StrataRpcImpl<D> {
+    async fn get_blocks_at_idx(&self, idx: u64) -> RpcResult<Vec<HexBytes32>> {
+        let l2_block_manager = self.l2_block_manager.clone();
+        let l2_blocks = l2_block_manager
+            .get_blocks_at_height_async(idx)
+            .await
+            .map_err(Error::Db)?;
+        let block_ids = l2_blocks
+            .iter()
+            .map(|id| HexBytes32(*id.as_ref()))
```

Comment:
We should add a `From` impl or even just `new` that makes this a little cleaner.